### PR TITLE
Set IRI version to major version only

### PR DESCRIPTION
**Pull request by RubenEschauzier**

### CODE REVIEW

**Comment by RubenEschauzier** [comment]
I see something is failing, will fix.

File: packages/jbr/lib/task/ExperimentLoader.ts [hunk 1]
```diff
@@ -23,8 +25,11 @@ export class ExperimentLoader {
   public static readonly COMBINATIONS_NAME = 'jbr-combinations.json';
   public static readonly PACKAGEJSON_NAME = 'package.json';
   public static readonly PREPAREDMARKER_PATH = [ 'generated', '.prepared' ];
-  public static readonly IRI_EXPERIMENT_HANDLER = `https://linkedsoftwaredependencies.org/bundles/npm/jbr/^${pJson.version}/components/experiment/ExperimentHandler.jsonld#ExperimentHandler`;
-  public static readonly IRI_HOOK_HANDLER = `https://linkedsoftwaredependencies.org/bundles/npm/jbr/^${pJson.version}/components/hook/HookHandler.jsonld#HookHandler`;
+  public static readonly IRI_EXPERIMENT_HANDLER = `https://linkedsoftwaredependencies.org/bundles/npm/jbr/` +
+  `^${coerce(major(pJson.version))!.raw}/components/experiment/ExperimentHandler.jsonld#ExperimentHandler`;
```
Comment (rubensworks) [inline packages/jbr/lib/task/ExperimentLoader.ts:29]: Can this not be simplified? E.g. https://github.com/LinkedSoftwareDependencies/Components-Generator.js/blob/69eefcef6606f3d256571edd2f6ff58232749e93/lib/parse/PackageMetadataLoader.ts#L37

Comment (RubenEschauzier) [reply packages/jbr/lib/task/ExperimentLoader.ts]: Yep added it

File: packages/jbr/package.json [hunk 2]
```diff
@@ -66,6 +66,7 @@
     "@types/cross-spawn": "^6.0.2",
     "@types/dockerode": "^3.2.3",
     "@types/fs-extra": "^9.0.11",
+    "@types/semver": "^7.5.1",
```
Comment (rubensworks) [inline packages/jbr/package.json:69]: Could you also add `semver` as dependency? Weird that the linter did not complain about this.

Comment (RubenEschauzier) [reply packages/jbr/package.json:69]: Good point, added it. Yeah, usually I just wait for the linter to complain about dependencies and add them.

**Comment by rubensworks** [comment]
Released as 3.0.2.

**Pull request merged**